{"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/1141/comments/10967164.json"}, "html": {"href": "#!/osrf/gazebo/issues/1141#comment-10967164"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/1141.json"}}, "type": "issue", "id": 1141, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "make doc is failing in Saucy/Trusty (doxygen problem)"}, "content": {"raw": "In ROS we are not even generating PDF files. The problem in our case is that the tagfiles generated when invoking doxygen on one package are continuously increasing in size. On every run doxygen adds duplicate entries to the generated tagfile. When using those to generate the documentation for other packages to achieve cross referencing the generated html contains duplicated content too.\n\nAt some point the tagfiles are even to big to be loaded by doxygen within two hours (the maximum time a doc job is allowed to run on our farm).\n\nI have tried 1.8.6 (Trusty) as well as 1.8.7 (Utopic) which both had that problem. Reverting to 1.7.6 (Precise) made it work again.", "markup": "markdown", "html": "<p>In ROS we are not even generating PDF files. The problem in our case is that the tagfiles generated when invoking doxygen on one package are continuously increasing in size. On every run doxygen adds duplicate entries to the generated tagfile. When using those to generate the documentation for other packages to achieve cross referencing the generated html contains duplicated content too.</p>\n<p>At some point the tagfiles are even to big to be loaded by doxygen within two hours (the maximum time a doc job is allowed to run on our farm).</p>\n<p>I have tried 1.8.6 (Trusty) as well as 1.8.7 (Utopic) which both had that problem. Reverting to 1.7.6 (Precise) made it work again.</p>", "type": "rendered"}, "created_on": "2014-06-28T23:52:50.762747+00:00", "user": {"display_name": "Dirk Thomas", "uuid": "{a8effe61-90f2-4949-89a8-9c6972071de7}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Ba8effe61-90f2-4949-89a8-9c6972071de7%7D"}, "html": {"href": "https://bitbucket.org/%7Ba8effe61-90f2-4949-89a8-9c6972071de7%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/2f77876936467715bd75258c967bcaa4d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsDT-2.png"}}, "nickname": "Dirk Thomas", "type": "user", "account_id": "557058:763d049c-c4c6-4e59-85a5-2d7a8fbbdd58"}, "updated_on": null, "type": "issue_comment", "id": 10967164}